pmd-core/src/main/java/net/sourceforge/pmd/renderers/AbstractAccumulatingRenderer.java

```diff
@@ -16,42 +24,73 @@
  * quite large in some scenarios. Consider using
  * {@link AbstractIncrementingRenderer} which can use significantly less memory.
  *
- * Subclasses should implement the {@link #end()} method to output the
- * {@link #report}.
- *
  * @see AbstractIncrementingRenderer
  */
 public abstract class AbstractAccumulatingRenderer extends AbstractRenderer {
 
-    /**
-     * The accumulated Report.
-     */
-    protected Report report;
```

Comment:
I don't think we can backport outputReport in a compatible way... Let's add this as an unresolved item to #3753, so that we don't forget to add this to the release notes for PMD 7, if we don't figure out a compatibility story until then